### DISCUSSION TITLE
Do not disconnect `pscale` CLI on CTRL+C

### DISCUSSION THREAD

**Discussion by twiddler** [planetscale/discussion#623] [Feature Requests] [answered]
In the CLI, I often find myself wanting to clear a command I was entering with <kbd>Ctrl</kbd>+<kbd>C</kbd>. That leads to either a disconnect or my terminal hanging in a weird limbo state. It'd be nice if it just cleared the current command I was about to send, and let me enter a new one.

(I would be perfectly fine with having to hit <kbd>Ctrl</kbd>+<kbd>C</kbd> twice to disconnect in those cases that I already started typing something.)

**Comment by mscoutermarsh** [comment, marked as answer]
Hey @twiddler,

Could you open an issue in the CLI repo? We'll look into it! https://github.com/planetscale/cli/issues

Thanks for the idea. And please let us know if you have any others while using the CLI.

**Reply by twiddler** [reply]
Sure thing, thanks for the link. 🙏

[Issue](https://github.com/planetscale/cli/issues/850)
